apps/studio/pages/project/[ref]/database/tables/index.tsx

```diff
@@ -48,7 +47,11 @@ const DatabaseTables: NextPageWithLayout = () => {
 
 DatabaseTables.getLayout = (page) => (
   <DefaultLayout>
-    <DatabaseLayout title="Database">{page}</DatabaseLayout>
+    <DatabaseLayout title="Database">
+      {/* <ProjectPageNavigation navKey="postgres-items"> */}
+      {page}
+      {/* </ProjectPageNavigation> */}
```

Comment:
Needed?
```suggestion
      {page}
```

Reply:
nope! ProjectPageNavigation doesn't seem to exist 🙏